views/remove-reason.html

```diff
@@ -35,9 +35,9 @@
 
         {% if errorList.length > 0 %}
```

Comment:
i think this block can be deleted as the variables being set aren't used in the rest of the file